build.sh

```diff
@@ -558,7 +558,8 @@ function clean_src_cpp_common()
 function check_flatData_zeroCopy_available()
 {
     version=$(awk -F"version" '/version/ { split($2, a, " "); print a[1] }' <<< $(${rtiddsgen_executable} -version)) # e.g. 3.0.0
-    ddsgen_version="${version//\./}" # e.g. 300
+    # We just need the Mayor value of the version.
+    ddsgen_version="$(echo $version | head -c 1)" # e.g. 3.0.5 -> 3
```

Comment:
If version is 10.0.0 for example, `echo $version | head -c 1 ` will take 1 and won't work. We should rather take all numbers until we reach a dot.

Reply:
```bash
major=`echo $version | awk -F. '{print $1}'` 
```
This should work

Reply:
Nice! Good catch Thanks!

Reply:
Done

Reply:
Thanks @salvacorts!!!